src/i18n.js

```diff
@@ -62,7 +62,21 @@ const resources = {
             errorMessage: "Error while sending review",
             successMessage: "Review succeeded"
         },
+        claimCreate: {
+            errorMessage: "Error while fetching claim",
+            responseC: "create with success",
+            errorMessage1: "Error while saving claim",
```

Comment:
Avoid to enumerate the filed, be more specific, for example `erroMessage1` should be `errorMessageSave`